Hymns/Audio Player/AudioPlayerView.swift

```diff
@@ -0,0 +1,85 @@
+import SwiftUI
+import AVFoundation
+import Combine
+
+enum PlaybackState: Int {
+    case waitingForSelection
+    case buffering
+    case playing
+}
+
+struct AudioView: View {
+
+    let player = AVPlayer()
+    let item: URL?
+    fileprivate let seekDuration: Float64 = 5
+    fileprivate var playerCurrentTime: Float64 {
+        CMTimeGetSeconds(self.player.currentTime())
+    }
+
+    @State var currentlyPlaying = true
+    @State var playbackState: PlaybackState = .waitingForSelection
+
+    var body: some View {
+        VStack {
+            AudioPlayerControlsView(player: player,
+                                    timeObserver: PlayerTimeObserver(player: player),
+                                    itemObserver: PlayerItemObserver(player: player), playbackState: $playbackState)
+            HStack(spacing: 30) {
+                //Reset button currently needed for when you tab to chords, piano, or guitar....
+                Button(action: {
+                    if self.currentlyPlaying {
+                        self.currentlyPlaying.toggle()
+                    }
+                    self.playbackState = .buffering
+                    self.player.replaceCurrentItem(with: nil)
```

Comment:
Instead of replacing the current item with nil then with itself, maybe we can just seek to the beginning

Reply:
It is easy to seek to 0 but then it does not truly reset the player for situations like when the song has ended or if you tab to guitar. So it works to just seek to 0 but not reset. 

I did remove the replacing the item with nil then itself to simply replace it with itself in commit 23f38cb This did give a tremendous performance boost to be almost instantaneous when you click the button now.